packages/react-examples/src/react-charting/VerticalBarChart/VerticalBarChartNegative.Example.tsx

```diff
@@ -0,0 +1,295 @@
+import * as React from 'react';
+import {
+  VerticalBarChart,
+  IVerticalBarChartProps,
+  IVerticalBarChartDataPoint,
+  ILineChartLineOptions,
+  DataVizPalette,
+  getColorFromToken,
+} from '@fluentui/react-charting';
+import { DefaultPalette } from '@fluentui/react/lib/Styling';
+import { IRenderFunction } from '@fluentui/react/lib/Utilities';
+import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
+import { Checkbox } from '@fluentui/react/lib/Checkbox';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+import { Label } from '@fluentui/react';
+
+interface IVerticalChartState {
+  width: number;
+  height: number;
+  isCalloutselected: boolean;
+  useSingleColor: boolean;
+  hideLabels: boolean;
+  showAxisTitles: boolean;
+  enableGradient: boolean;
+  roundCorners: boolean;
+}
+
+const options: IChoiceGroupOption[] = [
+  { key: 'basicExample', text: 'Basic Example' },
+  { key: 'calloutExample', text: 'Custom Callout Example' },
+];
+
+export class VerticalBarChartNegativeExample extends React.Component<IVerticalBarChartProps, IVerticalChartState> {
```

Comment:
this example isn't showing on the docsite.

Reply:
It must be visible. Can u try to restart